### ISSUE TITLE
Cadence CLI domain register requires global_domain option

### ISSUE THREAD

**Issue by sagikazarmark** [uber/cadence#2121]
Cadence version: 0.5.9

```
cadence --address 127.0.0.1 --domain play d re
Error: Option global_domain must be provided.
```

That seems to be a boolean flag though.

**Comment by sagikazarmark** [comment]
Also:

```
cadence --address 127.0.0.1:7933 --domain play d re --global_domain=false
Error: Register Domain operation failed.
Error Details: BadRequestError{Message: Invalid cluster name: }
('export CADENCE_CLI_SHOW_STACKS=1' to see stack traces)
```

**Comment by sagikazarmark** [comment]
This worked:
```
cadence --address 127.0.0.1:7933 --domain play d re --global_domain=false --ac "active"
```

Although in case of a single cluster, this feels weird.

**Comment by dmetzgar** [comment]
Fixed by #2172
